corehq/apps/hqadmin/management/commands/clone_domain.py

```diff
@@ -73,7 +73,8 @@ def handle(self, existing_domain, new_domain, **options):
             self.copy_fixtures()
 
         if self._clone_type(options, 'locations'):
-            self.copy_locations()
+            types_only = self._clone_type(options, 'types_only')
```

Comment:
Yeah, i thought about breaking it into two functions where the location types one returned the mapping that we could pass to the second, but that required sharing state between two functions, which we dont do now and would create other restrictions on how you could pass in arguments that we would need to guard against